src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useContext, useState } from "react";
+import {
+	EpisodesContext as Episodes
+} from "COMPONENTS/Episodes";
+
+export default function EpisodePreview(props){
+
+	// CONTEXT
+	//-----------------------------
+	const {
+		activeIndex,
+		episodeCount
+	} = useContext(Episodes);
+
+
+	// STATE
+	// ----------------------------
+	const [hidden, setHidden] = useState(true);
```

Comment:
OK. I ended up using the useState instead of the initialState object to pass the values and setter callbacks in with the context.provider. All working as expected:
- episode 0 is not hidden by default
- clicking thumbnail link sets the episode 0 preview hidden to be false
- clicking close or next sets preview hidden back to true
- prev/next changes the activeIndex and sets hidden to false for both the episode and the preview of that episode

The main thing remaining is having the prev/next anchors using video sources instead of anchor IDs. Struggling to find the best way to include index-1 source and index+1 source as part of the data mapping function in Episodes. Any thoughts on best approach? Will have another look tomorrow with fresh eyes